### ISSUE TITLE
Roadmap of 2022

### ISSUE THREAD

**Issue by ije** [ije/esm.sh#252]
### Roadmap List
- [x] use cloudflare worker as front layer 
- [ ] ~~preload deps~~
- [ ] ~~transform jsx/ts/tsx/vue/svelte for browsers from **github**~~
  ```js
  import "https://build.esm.sh/github.com/ije/ui/Button.tsx"  // Content-Type: application/javascript
  import "https://build.esm.sh/github.com/ije/ui/Button.tsx?jsx=preact"  // Content-Type: application/javascript
  import "https://build.esm.sh/github.com/ije/ui/Button.vue"  // Content-Type: application/javascript
  import "https://build.esm.sh/github.com/ije/ui/Button.vue?ssr"  // Content-Type: application/javascript
  import "https://build.esm.sh/github.com/je/ui/Button.vue?css"  // Content-Type: text/css
  import "https://build.esm.sh/github.com/ije/ui/Button.svelte"  // Content-Type: application/javascript
  import "https://build.esm.sh/github.com/ije/ui/Button.svelte?ssr"  // Content-Type: application/javascript
  import "https://build.esm.sh/github.com/ije/ui/Button.svelte?css"  // Content-Type: text/css
  ```

**Comment by aiotter** [comment]
Don't forget to pin this issue instead of #99!

**Comment by trhinehart-godaddy** [comment]
Does `preload deps` mean defining the deep imports up front? Also, have you considered putting those imports in the response header to preload as well to help reduce the waterfall?

**Comment by KyleJune** [comment]
I believe there are some issues related to imports and version pinning that would be good to get addressed. I added a few comments to this issue after attempting to upgrade from v85 to v89 of esm.sh today.

https://github.com/ije/esm.sh/issues/325

- interopDefault appears to not work correctly.
- esm.sh version pinning doesn't always work for dependencies.
- There was a recent regression, I started getting a type issue after upgrading from v85 to v89.
